### PR TITLE
Add delete camera subcommand

### PR DESCRIPTION
**Pull request by Crabzmatic**
Possibility do delete camera by id.

### CODE REVIEW

**Review by marek-nereca — APPROVED** [review]

**Comment by blaskoa** [comment]
More or less all commands are described in /sfcli/README.md so please add/modify this readme as well

File: sfcli/README.md [hunk 1]
```diff
@@ -217,6 +217,16 @@ Options:
   -?|-h|--help   Show help information
 ```
 
+##### camera delete
+
+Delete a camera.
```
Comment (blaskoa) [inline sfcli/README.md:222]: also modify README around the camera base command - line 164 https://github.com/innovatrics/smartface/pull/62/files#diff-332fba503dbd59be0810ab3dc97908becbd00ac684c6181818f19b49a2e91dc9R164

Comment (Crabzmatic) [reply sfcli/README.md:223]: done

File: sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        private ICamerasRepository Repository { get; }
+
+        public DeleteCameraCmd(ICamerasRepository repository)
+        {
+            Repository = repository;
+        }
+
+        protected virtual async Task OnExecuteAsync(IConsole console)
+        {
+            if (!CameraId.HasValue)
```
Comment (innoqazmlp) [inline sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:23]: Is not possible to use required attribute like in AddCameraCmd?

Comment (Crabzmatic) [reply sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:24]: yes, added

Comment (innoqazmlp) [reply sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:24]: then I think this condition can be removed and this should be handled internally by framework

Comment (Crabzmatic) [reply sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:24]: yeah, I was testing if this is really the case

Comment (innoqazmlp) [reply sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:24]: and maybe it is also sufficient to declare CameraId only as pure Guid

Comment (Crabzmatic) [reply sfcli/SmartFace.Cli/Commands/SubCamera/DeleteCameraCmd.cs:24]: yes, done, it works as expected

**Review by innoqazmlp — APPROVED** [review]

**Pull request merged**